Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3454.cs

```diff
@@ -0,0 +1,68 @@
+﻿#if APP
```

Comment:
These aren't needed if you aren't using a XAML file

Reply:
It is needed that the auto test is not crashed on the `SetAutomationPropertiesName`. I'll move them closer `AutomationProperties` to  to be more clear.